mw-webapp/vite.config.ts

```diff
@@ -30,7 +30,7 @@ export default defineConfig(() => {
 
   return {
     build: {outDir: "build"},
-    plugins: [react(), eslint(), viteTsconfigPaths()],
+    plugins: [react(), eslint({exclude: ["/virtual:/**"]}), viteTsconfigPaths()],
```

Comment:
why do we need `{exclude: ["/virtual:/**"]}`? Could you add a comment please?

Reply:
Because of this error - 
![image](https://github.com/tritonJS826/masters-way/assets/114860794/822ace0f-b299-4407-9918-6c83ad4895ba)

Found only this fix - https://github.com/storybookjs/builder-vite/issues/535#issuecomment-1507352550

Reply:
Ok, could you add a comment to the code to document it? With this link if possible. It will save time next for future developers.

Reply:
Done